modules/ROOT/pages/configure-acm.adoc

```diff
@@ -435,19 +435,19 @@ image::access-mapping.png[Access Mapping for CommunityApi]
 
 === Enable Guest User Registration as Members [[enable-guest-user-registration-as-members]]
 
-These steps enable guests to register as members in your ACM portal.
+These steps enable guests to register as members in your API Community Manager portal.
 
 ==== Add Self Register Component to Registration Page [[add-self-register-custom-component-to-register-page]]
 
-The self-registration Lightning component enables guest users to register themselves as members of your community. If you're not using any of the ACM out-of-the-box templates, use these steps to add the ACM-specific self-registration Lightning component.
+The self-registration Lightning component enables guest users to register themselves as members of your community. If you're not using any of the API Community Manager out-of-the-box templates, use these steps to add the self-registration Lightning component specific to API Community Manager.
```

Comment:
Oh the debates we have had over "out-of-box" and "out-of-the-box"!
Consider "If you're not using any of the templates supplied with API Community Manager, follow these steps to...."

Reply:
That's much better! Fixed.